<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AudioLabel</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/resources/owl.svg</normaloff>:/icons/resources/owl.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidgetMain">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListWidget" name="listWidgetAudioList"/>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayoutAudioMainArea" stretch="0,1,0,0">
        <item>
         <widget class="QLabel" name="labelCurrentAudioName">
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QOpenGLWidget" name="openGLWidgetAudioPlayShow"/>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSliderRateOfProcess">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutAudioOperationArea">
          <item>
           <widget class="QLabel" name="labelTimeTips">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacerLeft">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonPlayBackMode">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/resources/loop.svg</normaloff>:/icons/resources/loop.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonStop">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/resources/stop.svg</normaloff>:/icons/resources/stop.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonPre">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/resources/pre.svg</normaloff>:/icons/resources/pre.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="shortcut">
             <string>Up</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonBackward">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/resources/backward.svg</normaloff>:/icons/resources/backward.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="shortcut">
             <string>Left</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonPlayPause">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/resources/play.svg</normaloff>:/icons/resources/play.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="shortcut">
             <string>Space</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonForward">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/resources/forward.svg</normaloff>:/icons/resources/forward.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="shortcut">
             <string>Right</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonNext">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/resources/next.svg</normaloff>:/icons/resources/next.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="shortcut">
             <string>Down</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonVolume">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/resources/volume.svg</normaloff>:/icons/resources/volume.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>false</bool>
            </property>
            <property name="arrowType">
             <enum>Qt::NoArrow</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSliderVolume">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="value">
             <number>99</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelVolume">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacerRight">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpenRecently">
     <property name="title">
      <string>Open Recently</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuAutoSave">
     <property name="title">
      <string>Auto Save</string>
     </property>
     <addaction name="action0s"/>
     <addaction name="action60s"/>
     <addaction name="action120s"/>
     <addaction name="action180s"/>
     <addaction name="action300s"/>
    </widget>
    <addaction name="actionOpenJsonFile"/>
    <addaction name="actionOpenAudioFolder"/>
    <addaction name="menuOpenRecently"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="menuAutoSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGetStarted"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuStepTime">
     <property name="title">
      <string>Step Time</string>
     </property>
     <addaction name="action1s"/>
     <addaction name="action3s"/>
     <addaction name="action5s"/>
     <addaction name="action10s"/>
     <addaction name="action30s"/>
    </widget>
    <widget class="QMenu" name="menuPlayBackRate">
     <property name="title">
      <string>Play Back Rate</string>
     </property>
     <addaction name="action0_5"/>
     <addaction name="action1_0"/>
     <addaction name="action1_5"/>
     <addaction name="action2_0"/>
    </widget>
    <addaction name="menuStepTime"/>
    <addaction name="menuPlayBackRate"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open File...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpenAudioFolder">
   <property name="text">
    <string>Open Audio Folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action12">
   <property name="text">
    <string>12</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action0s">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0 s</string>
   </property>
  </action>
  <action name="action60s">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>60 s</string>
   </property>
  </action>
  <action name="action120s">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>120 s</string>
   </property>
  </action>
  <action name="action180s">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>180 s</string>
   </property>
  </action>
  <action name="action300s">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>300 s</string>
   </property>
  </action>
  <action name="actionUserDefined">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>User Defined</string>
   </property>
  </action>
  <action name="actionGetStarted">
   <property name="text">
    <string>Get Started</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpenAudioFiles">
   <property name="text">
    <string>Open Audio Files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpenJsonFile">
   <property name="text">
    <string>Open Json File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action0point25">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0.25</string>
   </property>
  </action>
  <action name="action0point5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0.5</string>
   </property>
  </action>
  <action name="action0point75">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0.75</string>
   </property>
  </action>
  <action name="action1point0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
  </action>
  <action name="action1point25">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1.25</string>
   </property>
  </action>
  <action name="action1point5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1.5</string>
   </property>
  </action>
  <action name="action1point75">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1.75</string>
   </property>
  </action>
  <action name="action2point0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2.0</string>
   </property>
  </action>
  <action name="action2point5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2.5</string>
   </property>
  </action>
  <action name="action5s">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5 s</string>
   </property>
  </action>
  <action name="action1s">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1 s</string>
   </property>
  </action>
  <action name="action3s">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3 s</string>
   </property>
  </action>
  <action name="action0_5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0.5</string>
   </property>
  </action>
  <action name="action0_75">
   <property name="text">
    <string>0.75</string>
   </property>
  </action>
  <action name="action1_0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
  </action>
  <action name="action1_25">
   <property name="text">
    <string>1.25</string>
   </property>
  </action>
  <action name="action1_75">
   <property name="text">
    <string>1.75</string>
   </property>
  </action>
  <action name="action1_5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1.5</string>
   </property>
  </action>
  <action name="action2_0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2.0</string>
   </property>
  </action>
  <action name="action2_5">
   <property name="text">
    <string>2.5</string>
   </property>
  </action>
  <action name="action10s">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10 s</string>
   </property>
  </action>
  <action name="action30s">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>30 s</string>
   </property>
  </action>
  <action name="actionCurrentItemOnce">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0 Current Item Once</string>
   </property>
  </action>
  <action name="actionCurrentItemInLoop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1 Current Item In Loop</string>
   </property>
  </action>
  <action name="actionSequential">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2 Sequential</string>
   </property>
  </action>
  <action name="actionLoop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3 Loop</string>
   </property>
  </action>
  <action name="actionRandom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4 Random</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
